aspnetcore/mvc/controllers/filters/sample/src/FiltersSample/Filters/LoggingAddHeaderFilter.cs

```diff
@@ -24,11 +25,9 @@ public void OnResultExecuting(ResultExecutingContext context)
 
         public void OnResultExecuted(ResultExecutedContext context)
         {
-            var headerName = "OnResultExecuted";
-            context.HttpContext.Response.Headers.Add(
-                headerName, new string[] { "ResultExecutedSuccessfully" });
-            _logger.LogInformation($"Header added: {headerName}");
-
+            // Can't add to headers because response has already begun.
+            _logger.LogInformation("OnResultExecuted");
```

Comment:
Please don't show code like this, the framework already logs this kind of information.